src/medCore/process/medAbstractProcess.h

```diff
@@ -21,6 +21,17 @@
 
 class medAbstractProcessPrivate;
 
+namespace DATAERROR
```

Comment:
The aim was: callin something like `DATAERROR::PIXELTYPE;` in `medAbstractProcess`, and not just PIXELTYPE. As you want. The question also is: where do i put that enum? It would be interesting to access it from medToolBox also.